editor/configs/crashesColumns.ts

```diff
@@ -32,6 +33,10 @@ export const crashesColumns = {
     label: "Speed limit",
     editable: true,
     inputType: "number",
+    inputOptions: {
+      validate: commonValidations.isNullableInteger,
+      min: { value: 1, message: "Speed limit cannot be negative" },
```

Comment:
Would you update this message to `Must be a positive integer`?

<img width="205" alt="Screenshot 2025-04-30 at 2 12 38 PM" src="https://github.com/user-attachments/assets/aeb8b053-4705-415d-a001-3c65688309f0" />